src/app/inventory/store/d1-item-factory.ts

```diff
@@ -48,6 +49,16 @@ export function processItems(
     if (createdItem !== null) {
       createdItem.owner = owner.id;
       result.push(createdItem);
+    } else {
+      // the item failed to be created for some reason. 3 things can currently cause this:
+      // an exception occurred, the item lacks a definition, or it lacks a name
+      // not all of these should cause the store to consider itself hadErrors.
+      // dummies and invisible items are not a big deal
+      const bucketDef = defs.InventoryBucket[item.bucketHash];
+      // if it's a named, non-invisible bucket, it may be a problem that the item wasn't generated
+      if (bucketDef.category !== BucketCategory.Invisible && bucketDef.displayProperties.name) {
```

Comment:
no clue if any of this is relevant or possible for d1

Reply:
Likewise